manager/allocator/network.go

```diff
@@ -1071,6 +1072,16 @@ func (a *Allocator) procTasksNetwork(ctx context.Context, onRetry bool) {
 	}
 }
 
+// IsBuiltInNetworkDriver returns whether the passed driver is an internal network driver
+func (a *Allocator) IsBuiltInNetworkDriver(name string) bool {
+	return a.netCtx.nwkAllocator.IsBuiltInDriver(name)
+}
+
+// PredefinedNetworks returns the list of predefined network structures for a given network model
+func PredefinedNetworks() []networkallocator.PredefinedNetworkData {
```

Comment:
Right now it could, but when I add a separate CNI backend it would be inappropriate which is why I included it here. The alternative would be to gate the caller (in `becomeLeader`) on the network model, which is what this PR was trying to minimise the need for.